src/components/screens/auth/components/IdpSelectorModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ModalSelector from 'react-native-modal-selector';
+import {StyleSheet} from 'react-native';
+import {Types} from 'onewelcome-react-native-sdk';
+
+interface Props {
+  identityProviders: Types.IdentityProvider[];
+  onSetSelectedIdp: (id: string) => void;
+  visible: boolean;
+  onCancel: () => void;
+}
+// The empty fragment is intended here to hide the default button that the ModalSelector generates by default
```

Comment:
Any idea why that is exactly? And is it an option to pass `{null}`, for example?